src/game_actor.cpp

```diff
@@ -254,12 +260,27 @@ void Game_Actor::ChangeEquipment(int equip_type, int item_id) {
 	if (item_id != 0) {
 		Main_Data::game_party->RemoveItem(item_id, 1);
 	}
+
+	const RPG::Item* item = ReaderUtil::GetElement(Data::items, GetWeaponId());
+	const RPG::Item* item2 = ReaderUtil::GetElement(Data::items, GetShieldId());
```

Comment:
Could you add a comment here that this unequips the other item when a two-handed item is equipped?